tests/test_flask_oidc.py

```diff
@@ -53,22 +53,22 @@ def request(self, path, method='GET', post_string='', **kwargs):
             token_info = {'active': False}
             if req_token in ['query_token', 'post_token']:
                 token_info['active'] = True
-                token_info['scope'] = 'openid'
+                token_info['scope'] = ['openid']
```

Comment:
Ref. https://tools.ietf.org/html/rfc7662 section 2.2, the optional scope property/claim in the introspection response is a string, so I won' take on this change.